lib/jekyll/commands/doctor.rb

```diff
@@ -161,7 +161,7 @@ def url_valid?(url)
         end
 
         def url_absolute(url)
-          return true if Addressable::URI.parse(url).absolute?
+          return true if url && Addressable::URI.parse(url.to_s).absolute?
```

Comment:
Should we just `return false unless url` instead?

Reply:
I think we should allow just `String` instances to be parsed..